tsconfig.json

```diff
@@ -1,7 +1,7 @@
 {
   "extends": "./tsconfig.build.json",
   "compilerOptions": {
-    "noEmit": true,
+    "noEmit": false,
```

Comment:
Would like to learn why this is `false` now. Can you clarify please?

Reply:
If you use typescript project references, you can't set `noEmit`. TS simply raises an error.

Reply:
@kedrzu `tsconfig.json` is used by the IDE (Webstorm, for the majority of the team) and this will output files into `src` folder.
Build scripts are configured to work with `tsconfig.build.json`. This needs to be double-checked, because this small change can easily mess up the whole repo setup.

That's basically the only reason we have 2 tsconfig files. One is for the IDE, and one is for actual build process.

Reply:
Please double-check this @kedrzu, as @Pavel910 mentioned, the `tsconfig.json` is used for IDEs, not within actual build processes.

Reply:
In which IDE there is automatic build with TSC? I personally use vscode and it doesn't work this way.

Reply:
@kedrzu please read my message again :)

Reply:
If this flag is `on`, it will make a mess of your project, if your tsconfig is configured to emit declarations:
![image](https://user-images.githubusercontent.com/3920893/147552585-9bb91ae0-05fe-4243-906e-18354020934e.png)

Reply:
Sorry, I must have missed it :)
In the latest change, I enabled `noEmit: true`, but disabled project references to rely purely on `paths`. 
It works fine for vscode and as far as I understand, @Pavel910 confirmed, that works good for webstorm as well.